main.tf

```diff
@@ -5,29 +5,56 @@ terraform {
     region  = "us-west-2"
     key     = "indent/terraform.tfstate"
   }
+}
 
+# Configure the Okta Provider
+provider "okta" {
+  org_name  = var.okta_prefix
+  base_url  = "okta.com"
+  api_token = var.okta_token
```

Comment:
this is the Number 1 thing here: we need to handle this gracefully: what happens when we delete the token?

Current thought is a runonce stage that sets up the `okta_app_oauth`; we can then read the necessary values from state in the bucket without having to refresh it (and run into API rejections from token revocation).